kong/db/migrations/core/010_210_to_211.lua

```diff
@@ -0,0 +1,80 @@
+
+local re_match = ngx.re.match
+
+
+local core_entities_to_clean = {
```

Comment:
I don't know whether this is planned for an extra migration, but I want to point out that this is not including custom plugin entities on the cleanup. 

Of those, I *think* the only ones that might need cleanup (because they have custom uniques) are `oauth2_credentials` / `authorization_codes` / `tokens`. Please double check that I didn't miss any.

If the cleanup is needed there, then the helper functions used on this module need to be put in a place where they can be shared with the plugin migrations.

Reply:
> If the cleanup is needed there, then the helper functions used on this module need to be put in a place where they can be shared with the plugin migrations.

`kong/db/migrations/operations/210_to_211.lua` would be a good choice